neptune/internal/structs/stack.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2019, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+class Stack(object):
```

Comment:
What's wrong with using a list? Why do we need this wrapper?

Reply:
IMO it looks better this way but I can use plain list if you want me to.

Reply:
I'd prefer less code than more.